File: results/java/MAXIMUM_LENGTH_PREFIX_ONE_STRING_OCCURS_SUBSEQUENCE_ANOTHER.cpp
int f_filled ( string s , string t ) {
  int count = 0 ;
  for ( int i = 0 ;
  i < t . length ( ) ;
  i ++ ) {
    if ( count == t . length ( ) ) break ;
    if ( t [ i ] == s [ count ] ) count ++ ;
  }
  return count ;
}

File: results/java_codex/MAXIMUM_LENGTH_PREFIX_ONE_STRING_OCCURS_SUBSEQUENCE_ANOTHER.java
int maxPrefix(string s, string t) {
        int count = 0;
        for (int i = 0; i < t.length(); i++) {
            if (count == t.length()) break;
            if (t.at(i) == s.at(count)) count++;
        }
        return count;
    }